models/group.go

```diff
@@ -222,6 +223,20 @@ func insertTargetIntoGroup(t Target, gid int64) error {
 	return nil
 }
 
+// UpdateTarget updates the given target information in the database.
+func UpdateTarget(target Target) error {
+	targetInfo := map[string]interface{}{
```

Comment:
The docs show [here](http://jinzhu.me/gorm/curd.html#update) that a `map[string]interface` must be used to update all changed attributes, even empty ones. I tested it out using a `models.Target` instance and that seems to be the case.

I just added another test for this situation. If you update [this line](https://github.com/rcutmore/gophish/blob/92f4ca6a90197dda9c1289a68656c7b41ace57fc/models/group.go#L233) to `err := db.Model(&target).Where("id = ?", target.Id).Updates(target).Error` the new test will fail.